senza/error_handling.py

```diff
@@ -103,6 +103,11 @@ def die_unknown_error(self, unknown_exception: Exception):
                             'content of {fn}'.format(e=unknown_exception,
                                                      fn=file_name))
 
+    # TODO :: sounds like I will have to do something here
+    # InvalidCiphertextException
+    # KeyUnavailableException
```

Comment:
I think those are going to be `ClientError` and we can skip them for now, since the only consequence is an ugly error message.

Reply:
Fair enough. I'll add the exception handling as a TODO in the issue.